README.md

```diff
@@ -72,7 +72,15 @@ npm install
 
 ```
 
-### 8. Run the development server.
+### 8. Create .env file and add the following env variables for the Vue app
+
+```
+VUE_APP_PDAP_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhcGlfa2V5IjoiNmVlOTQ2ZmI3YjY0NGUzNTgxNWVkMzMzOWY0MGRjNmQifQ.UMqlzKkJxwmz0bJy9BSONcoetg9zErl0XTkip1-W7a0
```

Comment:
Can we safely assume that committing this token publicly does not present a security risk?
And if so, why don't we just create a `.env.local` file or something and set package scripts to point to that in the test/local environment?

Reply:
That's true, I think .env file with just this in it makes sense

Reply:
Makes sense to me, too